### DISCUSSION TITLE
Is there any way to change the shortcut?

### DISCUSSION THREAD

**Discussion by ztalien** [Comfy-Org/ComfyUI#3057] [General]
For example, double click left button can open node quick search palette, but I want to change it to the "tab" button。

**Comment by subhasish-smiles** [comment]
I also need this. Please provide a way. Thank you!

**Comment by LukeG89** [comment]
@ztalien @subhasish-smiles With the latest update, you can add a keybind to open searchbox

![image](https://github.com/user-attachments/assets/eaa781d9-f92c-418d-a99c-647c9f4830f6)
